### PR TITLE
Update 0x04a-Mobile-App-Taxonomy.md

### PR DESCRIPTION
**Pull request by nick-epson**
Done, minus a final read with fresh eyes.

### CODE REVIEW

File: Document/0x04a-Mobile-App-Taxonomy.md [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
+The term "mobile app" refers to a self-contained computer program, designed to execute on a mobile device. Today, the Android and iOS operating systems cumulatively compose [more than 99% of the mobile OS market share] (http://www.idc.com/promo/smartphone-market-share/os). Additionally, mobile Internet usage has surpassed desktop usage for the first time in history, making mobile browsing and apps the [most widespread kind of Internet-capable applications] (http://www.idc.com/promo/smartphone-market-share/os), considering the market saturation of smart devices.
+
+> In this guide, we'll use the term "app" as a general term for referring to any kind of application running on popular mobile OSes.
+
+Throughout this guide, we will focus on apps for the two platforms dominating the market share: Android and iOS. Mobile devices are currently the most common item classification running these platforms – increasingly, the platforms (in particularly, Android) run on other devices, such as smartwatches, TVs, car navigation/audio systems, and other embedded systems.
```
Comment (b-mueller) [inline Document/0x04a-Mobile-App-Taxonomy.md]: in particularly -> in particular

Comment (b-mueller) [inline Document/0x04a-Mobile-App-Taxonomy.md]: dominating the market share -> dominating the market

File: Document/0x04a-Mobile-App-Taxonomy.md [hunk 2]
```diff
@@ -1,32 +1,40 @@
...
+The term "mobile app" refers to a self-contained computer program, designed to execute on a mobile device. Today, the Android and iOS operating systems cumulatively compose [more than 99% of the mobile OS market share] (http://www.idc.com/promo/smartphone-market-share/os). Additionally, mobile Internet usage has surpassed desktop usage for the first time in history, making mobile browsing and apps the [most widespread kind of Internet-capable applications] (http://www.idc.com/promo/smartphone-market-share/os), considering the market saturation of smart devices.
```
Comment (b-mueller) [inline Document/0x04a-Mobile-App-Taxonomy.md]: program, desgined -> is this comma needed?

Comment (b-mueller) [inline Document/0x04a-Mobile-App-Taxonomy.md]: compose -> comprise (?)

Comment (b-mueller) [inline Document/0x04a-Mobile-App-Taxonomy.md]: make sure that the references don't have a space in between e.g.

market share] (http:  -> market share](http:

File: Document/0x04a-Mobile-App-Taxonomy.md [hunk 3]
```diff
@@ -1,32 +1,40 @@
...
 
 ### Native App
 
-Most operating systems, including Android and iOS, come with a set of high-level APIs that can be used to develop applications specifically for that system. Such applications are called `native` for the system for which they have been developed. Usually, when discussing a `mobile app`, the assumption is that it is a `native app`, implemented in the standard programming languages for that operating system - either Objective-C or Swift for iOS, and Java or Kotlin for Android.
+Most operating systems, including Android and iOS, come with a set of high-level APIs used to develop applications, specific to the OS. Such applications are referred to as *native* to the system for which they have been developed. When discussing an app, the general assumption is that it is of this design, further implemented in a standard programming language for the respective operating system - either Objective-C or Swift for iOS, and Java or Kotlin for Android.
```
Comment (b-mueller) [inline Document/0x04a-Mobile-App-Taxonomy.md]: applications, specifc -> remove comma

File: Document/0x04a-Mobile-App-Taxonomy.md [hunk 4]
```diff
@@ -1,32 +1,40 @@
...
 
-There can be some ambiguity when discussing `native` apps for Android. Android provides two sets of APIs to develop against - the Android SDK and the Android NDK. The SDK (or Software Development Kit) is a Java API and is the default API against which applications are built. The NDK (or Native Development Kit) is a C/C++ based API used for developing application components that require specific optimization, or which can otherwise benefit from access to lower level APIs (such as OpenGL). Normally, you can only distribute apps built with the SDK, which potentially can also consume NDK APIs. Therefore we say that Android `native **apps**` (built with the SDK) can have `native **code**` (built with the NDK).
+Normally with Android, you can only distribute apps built with the SDK which can incorporate some developmental features from NDK APIs. Therefore, we say that Android *native apps* (i.e. built with the SDK) may have *native* code also built with the NDK. However, it should be noted that apps developed with the NDK API cannot incorporate SDK elements.
```
Comment (b-mueller) [inline Document/0x04a-Mobile-App-Taxonomy.md]: I'd remove this sentence.

"However, it should be noted that apps developed with the NDK API cannot incorporate SDK elements."

File: Document/0x04a-Mobile-App-Taxonomy.md (hunk 2 above)
Comment (nick-epson) [inline Document/0x04a-Mobile-App-Taxonomy.md]: Nope. I shuffled a lot of stuff around in the beginning and somehow that must have been left behind.

Comment (nick-epson) [inline Document/0x04a-Mobile-App-Taxonomy.md]: Ah, I see. Noted. Removed all the spaces.

File: Document/0x04a-Mobile-App-Taxonomy.md (hunk 1 above)
Comment (nick-epson) [inline Document/0x04a-Mobile-App-Taxonomy.md]: Removed "share".

File: Document/0x04a-Mobile-App-Taxonomy.md (hunk 4 above)
Comment (nick-epson) [inline Document/0x04a-Mobile-App-Taxonomy.md]: Removed.

**Comment by nick-epson** [comment]
All done.

**Pull request merged**